lib/signal_tower/prometheus_http_handler.ex

```diff
@@ -0,0 +1,11 @@
+defmodule SignalTower.PrometheusHTTPHandler do
```

Comment:
This is strange, that the httphandler is on another level than the promethus module. Since we're getting rid of the original stats module anyways, what about SignalTower.Prometheus and SignalTower.PrometheusHTTPHandler?
Another variant would be SignalTower.Prometheus.HTTPHandler but then we have an extra directory for only one file, I guess that doesn't make sense?

Reply:
That’s true :thinking: Probably the cleanest way in the future would be to just a common interface for stats and allow to configure the concrete stats module & setup tuple in environment configs (So, you could have e.g. csv file stats in dev, and Prometheus in the prod env, etc.). So if needed there could be a `stats.ex` and a `/stats` directory that holds all available different stats approaches.

But I think to keep it simple for now it makes sense to follow your suggestion. So, I’d keep the former module name and put `SignalTower.Prometheus` as `SignalTower.PrometheusStats` in `prometheus_stats.ex` on the same level as `prometheus_http_handler.ex`. Are you okay with this @farao?

Reply:
Now that I think of it again, if we have a common interface, then we could also keep the csv stats as you said and have at least all the stats modules under a common stats/ directory. Let's better do it like that.

Reply:
:+1: cool, then I’ll try to change it to this.

Reply:
@farao I thought a bit about it and tried a couple of things. In the end I think it’s probably the easiest, if we keep the `SignalTower.PrometheusStats` for this PR.

So, I moved `stats/prometheus.ex` back to `prometheus_stats.ex` for now. 

As far as I can see, the custom stats per environment setup will be way easier to do as soon as we changed to a DynamicSupervisor. Because then the stats config would just be another item in the children list (and could share the same format as opposed to what we currently have, see https://github.com/farao/signaltower/pull/4/files#diff-1418d77eb6456bbac161ceff0c15f276R38 and following lines).

Also these changes would get pulled in soonish and we can address the stats changes in another more focused stats PR.